modules/tinymce/src/themes/silver/test/ts/browser/skin/OxideColorSwatchMenuTest.ts

```diff
@@ -88,11 +100,25 @@ describe('browser.tinymce.themes.silver.skin.OxideColorSwatchMenuTest', () => {
   const closeSwatchButtonMenu = closeMenu('swatch-button');
   const openAndGetForecolorMenu = openAndGetMenu('Text color');
   const closeForecolorMenu = closeMenu('Text color');
+  const pOpenAndGetMenuColorMenu = async (editor: Editor) => {
+    const mainButton = 'button:contains("Color")';
+    const submenuButton = 'div[title="Background color"]';
+    TinyUiActions.clickOnMenu(editor, mainButton);
+    await TinyUiActions.pWaitForUi(editor, submenuButton);
+    TinyUiActions.clickOnUi(editor, submenuButton);
```

Comment:
It's probably a good idea to make it clear through the selector that this is a menu item. So maybe something like `[role="menu"] div[title="Background color"]`

Reply:
Good idea.